tests/ui/documents/test_documents_utils.py

```diff
@@ -370,6 +370,22 @@ def test_get_file_links(app):
         'preview': '/documents/1/preview/test.pdf'
     }
 
+    # extension in upper case
+    file['key'] = 'test.PDF'
+    assert utils.get_file_links(file, document) == {
+        'download': '/documents/1/files/test.PDF',
+        'external': None,
+        'preview': '/documents/1/preview/test.PDF'
+    }
+
+    # Check dot in filename
```

Comment:
Does it work if my file is named "test.pdf.part1.pdf"?

Reply:
Yes.